pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -276,9 +276,13 @@ def on_validation_end(self, trainer, pl_module):
         epoch = trainer.current_epoch
 
         # support structured results
-        if metrics.get('checkpoint_on') is not None:
+        if metrics.get('checkpoint_on', None) is not None:
```

Comment:
```suggestion
        if metrics.get('checkpoint_on') is not None:
```
I guess that None is default option...
https://www.programiz.com/python-programming/methods/dictionary/get